### PR TITLE
cpu/sam3: added DAC implementation

### PR DESCRIPTION
**Pull request by haukepetersen**
Tested this with my multimeter and works as expected to some extend. The only thing that I could observe, is that the DAC is not covering the whole voltage range from 0V to 3V3, but covers only the range from 0.5 to ~2.7V. Can't really find anything about this in the datasheet, so its not 100% clear to me if this is a configuration issue or just the way the DAC works...

### CODE REVIEW

**Comment by aabadie** [comment]
@haukepetersen, DAC is not supported yet in SAM3 :)

Please rebase

**Comment by haukepetersen** [comment]
rebased

**Comment by haukepetersen** [comment]
@haukepetersen, DAC is not supported yet in SAM3 :)

yes, I noticed, but only after editing the release notes. May bad.

**Comment by aabadie** [comment]
@haukepetersen, Murdock is complaining because of remaining whitespaces (blank line at end of file). Can you fix ?

**Comment by haukepetersen** [comment]
should be fixed now.

**Comment by haukepetersen** [comment]
@aabadie: would you mind to take a look again? Thx!

**Comment by aabadie** [comment]
Finally tested, works.

File: boards/arduino-due/include/arduino_pinmap.h [hunk 1]
```diff
@@ -132,6 +132,14 @@ extern "C" {
 #define ARDUINO_A11             ADC_LINE(13)
 /** @} */
 
+/**
+ * @name    Mapping of Ardunino DAC pins to RIOT DAC lines
```
Comment (aabadie) [inline boards/arduino-due/include/arduino_pinmap.h]: s/Ardunino/Arduino/

**Comment by haukepetersen** [comment]
yes, done.

**Comment by aabadie** [comment]
Then go!

**Pull request merged**